src/cppcache/src/LRUList.cpp

```diff
@@ -96,33 +102,38 @@ void LRUList<TEntry, TCreateEntry>::getLRUEntry(LRUListEntryPtr& result) {
 
 template <typename TEntry, typename TCreateEntry>
 typename LRUList<TEntry, TCreateEntry>::LRUListNode*
-LRUList<TEntry, TCreateEntry>::getHeadNode(bool& isLast) {
-  LOCK_HEAD;
+LRUList<TEntry, TCreateEntry>::getHeadNode(bool* isLast) {
```

Comment:
Good questions... Adherence to Google C++ Style Guide on references vs. pointers. I think I did this before writing my questions about the practice on the dev@geode. I can back this change out.

For reference, the guide dictates that references must be const and should be used for in variables only. For out variable you should use pointer.